OpenMEEG/src/TestHead.txt

```diff
@@ -26,6 +26,7 @@ function(TESTHEAD HEADNUM)
     set(DIPPOS                 ${MODELBASE}.dip)
     set(DIPPOS-SKULLSCALP      ${MODELBASE}-skullscalp.dip)
     set(PATCHES                ${MODELBASE}.patches)
+    set(EITPATCHES             ${MODELBASE}.EITpatches)
```

Comment:
a EIT patch has a surfacic extension. hence the 4th numeric column, this is the radius of the patch.
so do I rename it ?